app/renderer/js/main.ts

```diff
@@ -81,6 +82,7 @@ export class ServerManagerView {
   tabIndex: number;
   presetOrgs: string[];
   preferenceView?: PreferenceView;
+  sortableSidebar: SortableJS | null;
```

Comment:
```suggestion
  sortableSidebar?: SortableJS;
```

to avoid explicitly assigning `null`.